### PR TITLE
Cut selection flags for write and read cuts functions

### PR DESCRIPTION
**Pull request by Thuener**
Related with #778, this PR adds: 
- cut_selection flag to use or not the cut selection algorithm when reading the files: This will significantly reduce the time to load a file
- write_only_selected_cuts to write only the selected cuts: This will reduce the file size and reading time significantly

### CODE REVIEW

File: src/plugins/bellman_functions.jl [hunk 1]
```diff
@@ -650,36 +656,40 @@ function write_cuts_to_file(
...
+                        Dict(
+                            "realization" => i,
+                            "intercept" => intercept,
+                            "coefficients" => copy(cut.coefficients),
+                            "state" => copy(state.state),
+                        ),
+                    )
+                end
+            end
+        end
```
Comment (odow) [inline src/plugins/bellman_functions.jl:692]: Why is this swapped around?

Comment (odow) [reply src/plugins/bellman_functions.jl:692]: Oh, it's just the weird diff

File: src/plugins/bellman_functions.jl [hunk 2]
```diff
@@ -650,36 +656,40 @@ function write_cuts_to_file(
...
                         "intercept" => intercept,
                         "coefficients" => copy(cut.coefficients),
                         "state" => copy(state.state),
                     ),
                 )
             end
         end
+        for (i, theta) in enumerate(node.bellman_function.local_thetas)
+            for (cut, state) in zip(theta.cuts, theta.sampled_states)
+                if !write_only_selected_cuts || cut.constraint_ref !== nothing
```
Comment (odow) [inline src/plugins/bellman_functions.jl:676]: How about we keep the old indentation, and we do instead:
```suggestion
                if write_only_selected_cuts && cut.constraint_ref === nothing
                    continue
                end
```
I prefer fewer levels of indentation and early return/continue/break where possible.

**Review by odow — APPROVED** [review]
I'm happy if you're happy. You should have permission to merge.

We're missing a test, but I'm not too worried about that.

**Comment by Thuener** [comment]
I can create the test. Any recommendation on the test?
Just write and read and calculate the bound is enough?

**Pull request merged**